VndbSharp/VndbSharp.csproj

```diff
@@ -4,8 +4,6 @@
 	</PropertyGroup>
 	<PropertyGroup>
 		<TargetFrameworks>netstandard1.3;netstandard2.0</TargetFrameworks>
-		<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-		<RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
```

Comment:
This line is still required, the build is not failing with this removal because there is an existing `packages.lock.json` in the repo